libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -666,10 +669,10 @@ void YamlConfiguration::parsePortYaml(YAML::Node *portNode,
   checkRequiredField(&inputPortsObj, "id",
                      CONFIG_YAML_REMOTE_PROCESS_GROUP_KEY,
                      "The field 'id' is required for "
-                         "the port named '" + nameStr + "' in the YAML Config. If this port "
```

Comment:
There are linter failures. We have a formatter that can be used with intellij with a little work. May help avoid some of these issues. You're making changes that override our formatter. Would probably behoove us to apply formatters as a pre commit hook.